react/features/base/config/reducer.js

```diff
@@ -320,6 +318,10 @@ function _translateLegacyConfig(oldValue: Object) {
         newValue.e2ee.e2eeLabels = oldValue.e2eeLabels;
     }
 
+    if (oldValue.defaultRemoteDisplayName) {
```

Comment:
This is bogus, you need to only do this if the value is not set, aka, it's undefined. You need to check like above, if interface config exists and if it contains something.